crates/bevy_ecs/src/entity/mod.rs

```diff
@@ -195,7 +198,7 @@ impl Entity {
     /// specific snapshot of the world, such as when serializing.
     #[inline]
     pub const fn id(self) -> u32 {
```

Comment:
```suggestion
    pub const fn index(self) -> u32 {
```

Reply:
This is definitely a more invasive change, as the rename touches a lot of code. But I think its still worth doing, as it provides the right mental model for the field value. I recommend doing the rename with rust-analyzer, as merging this suggestion as-is will break a lot of code.